docs/index.asciidoc

```diff
@@ -0,0 +1,119 @@
+:plugin: bytes
+:type: filter
+
+///////////////////////////////////////////
+START - GENERATED VARIABLES, DO NOT EDIT!
+///////////////////////////////////////////
+:version: %VERSION%
+:release_date: %RELEASE_DATE%
+:changelog_url: %CHANGELOG_URL%
+:include_path: ../../../../logstash/docs/include
+///////////////////////////////////////////
+END - GENERATED VARIABLES, DO NOT EDIT!
+///////////////////////////////////////////
+
+[id="plugins-{type}s-{plugin}"]
+
+=== Bytes filter plugin
+
+include::{include_path}/plugin_header.asciidoc[]
+
+==== Description
+
+Parse string representations of computer storage sizes, such as "123 MB" or "5.6gb", into
+their numeric value in bytes.
+
+This plugin understands: 
+
+- bytes ("B")
+- kilobytes ("KB" or "kB")
+- megabytes ("MB", "mb", or "mB")
+- gigabytes ("GB", "gb", or "gB")
+- terabytes ("TB", "tb", or "tB")
+- petabytes ("PB", "pb", or "pB")
+
+[id="plugins-{type}s-{plugin}-examples"]
+==== Examples
+
+
+[cols="<,<",options="header",]
+|=======================================================================
+|Input string |Prefix system | Numeric value in bytes
+|40|`binary` or `decimal`|40
```

Comment:
the doc below states that this can be `binary` or `metric` ... but the doc here says `binary` or `decimal`

Reply:
Yeah, this should be `metric`. Good catch! Fixing.